static/app/components/forms/formPanel.tsx

```diff
@@ -62,12 +62,21 @@ function FormPanel({
   const handleCollapseToggle = useCallback(() => setCollapse(current => !current), []);
 
   return (
-    <Panel id={typeof title === 'string' ? sanitizeQuerySelector(title) : undefined}>
+    <Panel
+      id={typeof title === 'string' ? sanitizeQuerySelector(title) : undefined}
+      onClick={collapsible ? handleCollapseToggle : undefined}
+      style={collapsible ? {cursor: 'pointer'} : undefined}
```

Comment:
Since Panel is a `div`, I'd add a `role="button"` when collapsible is set. Maybe `aria-label` and `aria-expanded` as well.

Reply:
good callout, thanks @leeandher 🙏 
updated it with your suggestions